tools/sigclient/pkg/ingest/ingest.go

```diff
@@ -348,7 +348,7 @@ func getReaderFromArgs(iType IngestType, nummetrics int, gentype string, str str
 	case "file":
 		log.Infof("Initializing file reader from %s", str)
 		rdr = utils.InitFileReader()
-	case "benchmark":
+	case "benchmark", "functional":
```

Comment:
have a separate case than benchmark, even though the code for now is the exact same, but we will change the `rdr` for functional

Reply:
addressed in the latest commit.